.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: true
+os: trusty
+cache: false
+env:
+  matrix:
+  - URL=https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz
+before_install:
+- openssl aes-256-cbc -K $encrypted_31db552cf6d7_key -iv $encrypted_31db552cf6d7_iv
```

Comment:
You can also consider using a docker image with SFDX preinstalled, that might reduce the build time a bit